app/code/Magento/Catalog/Test/Mftf/Section/StorefrontProductPageSection.xml

```diff
@@ -12,6 +12,7 @@
         <element name="qtyInput" type="button" selector="input.input-text.qty"/>
         <element name="addToCartBtn" type="button" selector="button.action.tocart.primary" timeout="30"/>
         <element name="successMsg" type="button" selector="div.message-success"/>
+        <element name="errorMsg" type="button" selector="div.message-error"/>
```

Comment:
Please DO NOT make edits outside of the "Inventory*" modules because non-"Inventory*" modules are NOT copied when you install MSI in Jenkins. I've replace the reference to the element with a waitForPageLoad function and the Test passes.